src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -353,7 +361,9 @@ export class PanelComponent extends Disposable implements IThemable {
 
 	private focusOnTabHeader(index: number): void {
 		if (index >= 0 && index <= this._tabHeaders.length - 1) {
-			this._tabHeaders.toArray()[index].focusOnTabHeader();
+			const tabHeaderArray = this._tabHeaders.toArray();
```

Comment:
if you need a specific element of the array, you can do 
```
const tabHeader = this._tabHeaders.toArray()[index]
```

Reply:
I was just saving it in a var so that I can use it multiple times.